EIPS/eip-eof-tracing.md

```diff
@@ -0,0 +1,173 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7756
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.